### DISCUSSION TITLE
Load a PlanetScale dump into a non-PlanetScale MySQL database

### DISCUSSION THREAD

**Discussion by phbernard** [planetscale/discussion#575] [Q&A] [answered]
I'm looking for a reliable way to load a PlanetScale dump (got from `pscale database dump`) into a non-PlanetScale MySQL database

I used to use `mysqldump` to dump my database. The output file can be easily and reliably loaded into a local MySQL database. However, `pscale database dump` generate several files (at least two for each table), so loading them is not straightforward.

Loading such a dump seems like a legit concern. After all, one wants to be able to use the data outside of PlanetScale if needed.

**Comment by jgreet** [comment, marked as answer]
`pscale db dump` creates dumps that are compatible with the backup and restore tool mydumper/myloader. The best way to restore a pscale dump to another MySQL instance is probably to use myloader: https://github.com/mydumper/mydumper

Alternatively, you could pipe every `.sql` file in to the the new database or concatenate them all in to to one big dump file similar to mysqldump. Just be sure a table's `-schema.sql` file goes before the same table's data files. That should happen by default if the dump files are sorted alphabetically. 

```sh
cat pscale_dump_folder/*.sql | mysql dbname
```

```sh
cat pscale_dump_folder/*.sql > onelargefile.sql
```

**Reply by phbernard** [reply]
I've just tested MyDumper, works like a charm. Thank you @jgreet!
